lib/yabeda/sidekiq.rb

```diff
@@ -46,6 +48,10 @@ module Sidekiq
         sidekiq_active_processes.set({}, stats.processes_size)
         sidekiq_jobs_retry_count.set({}, stats.retry_size)
 
+        ::Sidekiq::Queue.all.each do |queue|
+          sidekiq_jobs_latency.set({ queue: queue.name }, queue.latency)
```

Comment:
I think we have to round the seconds value for better visibility 🤔

Reply:
I don't think we actually need any rounding